src/main/resources/assets/ebwizardry/texts/handbook_zh_cn.json

```
@@ -492,11 +501,26 @@
 				"#recipe wizard_leggings",
 				"#recipe wizard_boots",
 
-				"你可以用附魔台为巫师护具附魔,而因为魔法绸缎的固有属性，它们倾向于被附魔上能够使它们留存的更久的附魔。一些附魔，魔法，冰霜，雷电保护附魔，对巫师具有特殊的实用价值，这些附魔可以有效抵御其它巫师对你发起的进攻。",
+				"你可以用附魔台为巫师护具附魔,而因为魔法绸缎的固有属性，它们倾向于被附魔上能够使它们留存的更久的附魔。一些附魔，例如魔法，冰霜，雷电保护附魔，对巫师具有特殊的实用价值，这些附魔可以有效抵御其它巫师对你发起的进攻。",
```

|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
				"你可以用附魔台为巫师护具附魔，而因为魔法绸缎的固有属性，它们倾向于得到能使它们留存更久的附魔。一些附魔，例如魔法、冰霜、雷电保护附魔，对巫师具有特殊的实用价值，这些附魔可以有效抵御其他巫师对你发起的进攻。",
```

---

```suggestion
				"你可以用附魔台为巫师护具附魔，而因为魔法绸缎的固有属性，它们倾向于得到能使它们留存更久的附魔。一些附魔，例如魔法、冰霜、雷电保护，对巫师具有特殊的实用价值，这些附魔可以有效抵御其他巫师对你发起的进攻。",
```